contracts/svm/testing/src/helpers.rs

```diff
@@ -56,16 +54,15 @@ pub fn warp_to_unix(svm: &mut litesvm::LiteSVM, unix_timestamp: i64) {
     svm.set_sysvar(&clock);
 }
 
-pub fn assert_custom_error(error: TransactionError, instruction_index: u8, custom_error: u32) {
+pub fn assert_custom_error(
+    error: TransactionError,
+    instruction_index: u8,
+    custom_error: InstructionError,
```

Comment:
maybe we could rename this `instruction_error`? since it's no longer just custom errors, it includes like `ArithmeticOverflow` and the like